UselessMine.cpp

```diff
@@ -455,17 +482,39 @@ bool UselessMine::SlashCommand(int playerID, bz_ApiString command, bz_ApiString
             bz_sendTextMessage(BZ_SERVER, playerID, "Death messages reloaded");
             return true;
         }
-        else if (params->size() == 0)
+        if (params->get(1) == "defusalmessages")
```

Comment:
Use an `else if` here. It'll allow for this section to short circuit once some condition is matched